docs/source/conf.py

```diff
@@ -109,7 +109,7 @@
 
 # -- Generate documentation -------------------------------------------------
 
-examples_dir = os.path.join(root_path, "tests", "tutorials")
+examples_dir = os.path.join(root_path, "tests/cmake_test", "tutorials")
```

Comment:
We should let `os.path.join` determine the path separators here:
```suggestion
examples_dir = os.path.join(root_path, "tests", "cmake_test", "tutorials")
```